src/core/agent/modules/generalStorage/repositories/identifierMetadataRecord.ts

```diff
@@ -10,7 +10,9 @@ interface IdentifierMetadataRecordProps {
   createdAt?: Date;
   isArchived?: boolean;
   isDeleted?: boolean;
+  isPending?: boolean;
   theme: number;
+  opName?: string;
```

Comment:
Rename this to `signifyOpName` so that it is obvious it isn't used with `did:key`